src/Toast/index.tsx

```diff
@@ -20,12 +19,30 @@ interface ToastAction {
 }
 
 interface ToastProps {
+  /** A string or an element that is rendered inside the main body of the `Toast`. */
   children: string;
+  /**
+   * A function that is called on close. It can be used to perform
+   * actions upon closing of the `Toast`, such as setting the "show"
+   * element to false.
+   * */
   onClose: () => void;
+  /** Boolean used to control whether the `Toast` shows */
```

Comment:
[optional]: Please add a period at the end, similar the previous descriptions.

```suggestion
  /** Boolean used to control whether the `Toast` shows. */
```